style.css

```diff
@@ -98,6 +98,55 @@
     opacity: 1;
   }
 
+  .modal-pop {
+    display: none;
+    height: 100vh;
+    position: fixed;
+    top: 0;
+    right: 0;
+    left: 0;
+    z-index: 10;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: flex-start;
+    background-color: #fff;
+    margin: 0;
+    box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+    overflow: auto;
+    padding: 30px;
+  }
```

Comment:
Kindly update your popup window on desktop screens to match the figma design better🙏

Your design:
<img width="466" alt="image" src="https://github.com/Armynerh/personal_portfolio/assets/37116322/b8b9c1a1-0e28-4b43-bae4-de42e6a35a1f">

Figma design:
<img width="428" alt="image" src="https://github.com/Armynerh/personal_portfolio/assets/37116322/ffbfdaf5-51f7-421f-89d5-69cf6ebbefe2">

A few things to note:
- Kindly update your modal so that it has the proper margins and the grey background shown in the figma design
- Kindly update your close button so it has the same styling as the figma design
- Kindly reduce the padding applied to your modal buttons and make sure the button icons are showing

-